src/Appwrite/Utopia/Request.php

```diff
@@ -9,48 +9,20 @@
 
 class Request extends UtopiaRequest
 {
-    /**
-     * @var Filter
-     */
-    private static $filter = null;
-
-    /**
-     * @var Route
-     */
-    private static $route = null;
+    private static ?Filter $filter = null;
+    private static ?Route $route = null;
 
-    /**
-     * Request constructor.
-     */
     public function __construct(SwooleRequest $request)
     {
         parent::__construct($request);
     }
 
     /**
-     * Get Params
-     *
-     * Get all params of current method
-     *
-     * @return array
+     * @inheritdoc
      */
     public function getParams(): array
     {
-        $requestParameters = [];
-
-        switch ($this->getMethod()) {
-            case self::METHOD_GET:
-                $requestParameters = (!empty($this->swoole->get)) ? $this->swoole->get : [];
-                break;
-            case self::METHOD_POST:
-            case self::METHOD_PUT:
-            case self::METHOD_PATCH:
-            case self::METHOD_DELETE:
-                $requestParameters = $this->generateInput();
-                break;
-            default:
-                $requestParameters = (!empty($this->swoole->get)) ? $this->swoole->get : [];
-        }
+        $requestParameters = parent::getParams();
```

Comment:
```suggestion
        $parameters = parent::getParams();
```

Request is already part of the namespace, we shouldn't reuse it.